tests/RepositoryTest.php

```diff
@@ -172,12 +172,10 @@ public function testAll()
         $this->assertCount(1, $users);
     }
 
-    public function testAllSQLQuery()
+    public function testAllSQL()
     {
         $repository = $this->repository(User::class);
-        $users      = $repository->all(
-            new SQLQuery($this->connection(), $repository->select()->orderBy('id DESC')->getSelectQuery())
-        );
+        $users      = $repository->sql($repository->select()->orderBy('id DESC')->getSelectQuery())->all();
```

Comment:
These tests could be improved by actually passing the SQL instead of using the query builder.

```suggestion
        $users      = $repository->sql("SELECT * FROM `user` ORDER BY id DESC")->all();
```